disperser/apiserver/server.go

```diff
@@ -609,18 +609,23 @@ func (s *DispersalServer) validateRequestAndGetBlob(ctx context.Context, req *pb
 		return nil, err
 	}
 
+	requiredQuorums, err := s.tx.GetRequiredQuorumNumbers(ctx)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Not strictly necessary, but it would be nice to cache the required quorums just like we're caching the quorum count using `updateQuorumCount` method.
It may be a small change to reuse/update existing logic in `updateQuorumCount` to cache them. 
We can leave a TODO and handle it later though

Reply:
Hmm, interesting. I agree that it would be nice to cache this. But is it possible to use the same pattern? For the `quorumCount` we can detect if a new quorum has been added, if a request contains a quorum which is greater than the `quorumCount`. This wouldn't work for the `requiredQuorums`.

Reply:
Ah that's a good point. It would be tricky to cache `requiredQuorums`

Reply:
From a robustness standpoint, we could store the state of these variables and fallback to that state if the RPC was unavailable. 

In the worst case, this could result in a blob failing to verify (a required quorum was added while RPC was down). 

I think this is worthwhile.